src/main/scala/beam/agentsim/agents/modalbehaviors/ChoosesMode.scala

```diff
@@ -799,6 +702,28 @@ trait ChoosesMode {
         )
   } using completeChoiceIfReady)
 
+  private def correctCurrentTripModeAccordingToRules(
+    personData: BasePersonData,
+    nextAct: Activity,
+    availableModes: Seq[BeamMode]
+  ): Option[BeamMode] = {
+    val replanningIsAvailable =
+      personData.numberOfReplanningAttempts < beamServices.beamConfig.beam.agentsim.agents.modalBehaviors.maximumNumberOfReplanningAttempts
+    (personData.currentTripMode, personData.currentTourMode) match {
+      case (_, Some(CAR | BIKE)) if personData.currentTourPersonalVehicle.isDefined => personData.currentTourMode
```

Comment:
This seems also a new behavior - as long as tour/trip aren't getting confused....I guess this raises the question of naming - should a different name be used so somebody doesn't accidentally use the wrong variable.

Reply:
You are right. @zneedell What do you think about `currentTourMode` vs `currentTripMode` names?